scene/resources/packed_scene.cpp

```diff
@@ -249,6 +270,31 @@ Node *SceneState::instantiate(GenEditState p_edit_state) const {
 			// Get the node from somewhere, it likely already exists from another instance.
 			if (parent) {
 				node = parent->_get_child_by_name(snames[n.name]);
+				if (i < ids.size()) {
+					if (!node) {
+						Node *base = parent;
+						while (base != ret_nodes[0] && base->get_scene_file_path() == "") {
```

Comment:
```suggestion
						while (base != ret_nodes[0] && !base->is_instance()) {
```
New method for checking whether a Node is an instance (it's effectively the same, but it should be preferred).

Reply:
Fixed in 7e1e7ab. I also moved this further down after the warning because it's only used in that branch.